Formula/kubevela.rb

```diff
@@ -0,0 +1,31 @@
+class Kubevela < Formula
+  desc "Application Platform based on Kubernetes and Open Application Model"
+  homepage "https://kubevela.io"
+  url "https://github.com/oam-dev/kubevela.git",
+    using:    :git,
+    tag:      "v0.2.0",
+    revision: "79c2209d19ddc62bb6682c91b3a485b4f939ab5f"
+  license "Apache-2.0"
+
+  depends_on "go" => :build
+  depends_on "git"
```

Comment:
It's this needed by the application? Can't it use the macOS one?

Reply:
sorry